spark-statsd-reporter/src/main/java/org/apache/spark/metrics/sink/statsd/InstanceDetailsProvider.java

```diff
@@ -38,6 +38,7 @@ InstanceDetails buildInstanceDetails() {
         return new InstanceDetails(
                 sparkConf.getAppId(),
                 sparkConf.get("spark.app.name"),
+                System.getenv("SPARK_APP_ORIGIN"),
```

Comment:
let's use verbose naming of environment variables as per PR https://github.com/mesosphere/spark-build/pull/526

Reply:
changed as suggested